version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
```

Comment:
I work on an [art program](https://github.com/mypaint), and I'm interested in growing its community and keeping working on the code a non-sucky experience for everyone. This question is relevant to us in odd ways:

* Certain standard test images I will not use as a matter of principle, and I would expect contributors not to within our project spaces (ref. Wikipedia: "Lenna#Controversy", page contains a cropped part of a sexualized image above the referenced section, just the woman's head)
* We require screenshots of the program in operation, for publicity and for debugging glitches. Those obv. have to be SFW (and safe for kids), but threads designed to gather them may by accident not always be (artist-testers should not be not be bound by our CoC outside our project spaces). The practical way of handling an accidental overstep should be to replace with a link or delete, with an appropriate request to refrain or a warning. Fine where we have control of course...
* I want artist-testers to be able to draw whatever the hell they want in spaces coded for that sort of thing. Our CoC should expect them not to use that freedom to harass without consequence, but it should also accept that some non-project spaces are sex-positive, kink-friendly, provide sex ed., or are just artist hangouts where folks might talk about the neat programs they use or hack on in the same breath as talking about the art they make.

I vacillate on this so much. I do not like the word "overt" as a replacement. May I propose

> * Inappropriate sexual language or imagery. Within this project's spaces, sexual language or imagery of any sort is never appropriate.

as a replacement? The implication being that outside project spaces, interpretation hinges on what is appropriate - there. And that directed harassment as interpreted anywhere is never appropriate, full stop.